src/lib/select/select.spec.ts

```diff
@@ -4404,3 +4439,44 @@ class SingleSelectWithPreselectedArrayValues {
   @ViewChild(MatSelect) select: MatSelect;
   @ViewChildren(MatOption) options: QueryList<MatOption>;
 }
+
+@Component({
+  selector: 'select-without-option-centering',
+  template: `
+    <div [style.height.px]="heightAbove"></div>
+    <mat-form-field>
+      <mat-select placeholder="Food" [formControl]="control" [required]="isRequired"
+        [tabIndex]="tabIndexOverride" [aria-label]="ariaLabel" [aria-labelledby]="ariaLabelledby"
+        [panelClass]="panelClass" [disableRipple]="disableRipple" disableOptionCentering=true>
+        <mat-option *ngFor="let food of foods" [value]="food.value" [disabled]="food.disabled">
+          {{ food.viewValue }}
+        </mat-option>
+      </mat-select>
+    </mat-form-field>
+    <div [style.height.px]="heightBelow"></div>
+  `
+})
+class SelectWithoutOptionCentering {
```

Comment:
Can you trim this component down only to the functionality necessary to test the new option? Alternatively you should be able to re-use some of the other test components.

Reply:
I've gone with trimming the component down and got rid of what I could. The `@Input` and unit test description have also been updated.